website/content/partials/telemetry-metrics/vault/replication/rpc/server/persist_persona_request.mdx

```diff
@@ -0,0 +1,5 @@
+### vault.replication.rpc.server.persist_persona_request ((#vault-replication-rpc-server-persist_persona_request))
+
+Metric type | Value | Description
+----------- | ----- | -----------
+summary     | ms    | Time required to complete a request to persist a persona
```

Comment:
```suggestion
summary     | ms    | Time required to complete a request to persist an alias
```

Persona is the old terminology.

Reply:
I think this one might be obsolete, from what I can tell we won't emit it these days.

Reply:
Okay. I'll commit the suggestion for now, but won't delete it from the metrics listings unless you confirm it's no longer populated.